README.md

```diff
@@ -131,7 +135,8 @@ Citation
 ---------
 If you use pyGenomeTracks in your analysis, you can cite the following paper :
 
-Fidel Ramírez, Vivek Bhardwaj, Laura Arrigoni, Kin Chung Lam, Björn A. Grüning, José Villaveces, Bianca Habermann, Asifa Akhtar & Thomas Manke. High-resolution TADs reveal DNA sequences underlying genome organization in flies. Nature Communications (2018) [doi:10.1038/s41467-017-02525-w](https://www.nature.com/articles/s41467-017-02525-w)
+Lopez-Delisle L, Rabbani L, Wolff J, Bhardwaj V, Backofen R, Grüning B, Ramírez F, Manke T. pyGenomeTracks: reproducible plots for multivariate genomic data sets. Bioinformatics. 2020 Aug 3:btaa692. [doi: 10.1093/bioinformatics/btaa692](https://doi.org/10.1093/bioinformatics/btaa692). Epub ahead of print. PMID: 32745185.
+
```

Comment:
I would not replace the old citation. Just mention both.

Reply:
ok

Reply:
So I ask user to cite both? (Personally I only cited the last one in my last publication but it is fine to put both in the next one).

Reply:
Yes, we ask them to cite both; if they do it, is something else.

Reply:
ok done